plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -2,8 +2,64 @@
  * External dependencies
  */
 import { store } from '@woocommerce/interactivity';
+import { HTMLElementEvent } from '@woocommerce/types';
+
+const getInputData = ( event: HTMLElementEvent< HTMLButtonElement > ) => {
+	const target = event.target as HTMLButtonElement;
+
+	const inputElement = target.parentElement?.querySelector(
+		'.input-text.qty.text'
+	) as HTMLInputElement | null | undefined;
```

Comment:
Should we try to make it interactive-friendly by relying on the API attributes instead of a CSS class name?
Something that we can polish later in case it makes sense, tho.

Reply:
Yes, good point. I don't think the input element has any Interactivity API attributes right now. Do you think we should add any? Otherwise, I think using `input[name="quantity"]` might be more semantic than the CSS class names. How would that sound?

(I'm going to leave it as-is in this PR, given that this is the same code as from the legacy Add to Cart form block, but happy to change it in a follow-up)